notebooks/WET/WET-007/WET-007_20241028_tie-vs-danish.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
<p>typo "mostly gree". </p><p> </p><p>And I think it strengthens the text by putting a numerical value ("noisiest,  with 150nm rms"), and later about disagreements (~200 nm difference for Z5, with amplitude of ~1200 nm is ~16% , and for Z13,Z20...)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Done